packages/protocol/contracts/governance/Governance.sol

```diff
@@ -1052,4 +1053,50 @@ contract Governance is IGovernance, Ownable, Initializable, UsingLockedGold, Ree
     }
     return threshold;
   }
+
+  // call has been separated into its own function in order to take advantage
+  // of the Solidity's code generator to produce a loop that copies tx.data into memory.
+  /**
+   * @notice Executes a function call.
+   * @param value The value of Celo Gold to be sent with the function call.
+   * @param destination The destination address of the function call.
+   * @param dataLength The length of the data to be included in the function call.
+   * @param data The data to be included in the function call.
+   */
+  function externalCall(
+    address destination,
+    uint value,
+    uint dataLength,
+    bytes memory data
+  )
+    private
+    nonReentrant
+    returns (bool)
+  {
+    bool result;
+
+    require(AddressesHelper.isContract(destination), "Invalid contract address");
```

Comment:
Do we want to always require this? Or only when dataLength is > 0?

I would imagine in some cases we will have governance proposals to send gold or dollars to an EOA.